docs/hydra/faq.md

```diff
@@ -10,6 +10,43 @@ removes or updates it!**
 
 <!-- toc -->
 
+## How to use Ory Hydra with AWS CloudHSM
```

Comment:
Maybe move this to HSM docs directly?

Reply:
I mean adding it here: https://www.ory.sh/docs/hydra/guides/hsm-support

Reply:
Oh my bad, I totally missed that document 🙈 
Will add it there thanks!